packages/docs/docs/advanced/custom-font.mdx

```diff
@@ -0,0 +1,71 @@
+# Custom font
+
+To use a font from hosters like Google Fonts. First make a css file under `src`
+
+```
+root
+└─src
+  ├─scenes/
+  ├─motion-canvas.d.ts
+  ├─project.meta
+  ├─project.ts
+  └─global.css *
+```
```

Comment:
You can the `diff` language to better highlight the changes:
````suggestion
```diff
root
└─src
  ├─scenes/
  ├─motion-canvas.d.ts
  ├─project.meta
  ├─project.ts
+ └─global.css
```
````
![image](https://user-images.githubusercontent.com/64662184/216876633-818f7714-c603-4785-843f-ba6e5505e2bd.png)